ncm-ssh/src/main/perl/ssh.pm

```diff
@@ -20,20 +20,25 @@ our $NoActionSupported = 1;
 
 use CAF::Process;
 use CAF::FileEditor;
-use constant SSHD_CONFIG  => "/etc/ssh/sshd_config";
-use constant SSH_CONFIG   => "/etc/ssh/ssh_config";
-use constant SSH_VALIDATE => qw(/usr/sbin/sshd -t -f /proc/self/fd/0);
+use constant DEFAULT_SSHD_CONFIG  => "/etc/ssh/sshd_config";
+use constant DEFAULT_SSHD_PATH    => "/usr/sbin/sshd";
+use constant DEFAULT_SSH_CONFIG   => "/etc/ssh/ssh_config";
+use constant SSHD_CONFIG_MULTILINE => qw(HostKey AcceptEnv ListenAddress);
```

Comment:
`AccetpEnv` is currenlty a boolean, which makes no sense at all. nice catch making this work.